test/e2e/webdriver/driver.js

```diff
@@ -80,13 +80,13 @@ function wrapElementWithAPI(element, driver) {
       await element.originalClick();
     } catch (e) {
       if (e.name === 'ElementClickInterceptedError') {
-        if (e.message.includes('<div class="mm-box loading-overlay">')) {
+        if (e.message.includes('<div class="mm-box loading-overlay"')) {
```

Comment:
the test was not failing because of this, but taking the opportunity to also fix this guard. If we close the div, we'll never match the error message, since usually they take the form of:


`<div class="..." style="." ...>`